docs/dashboards/interactive.md

```diff
@@ -42,7 +42,7 @@ For questions composed using the query builder, you can select from three option
 - Go to a custom destination.
 - Update a dashboard filter (if the dashboard has a filter).
 
-SQL questions will only have the option to **Go to a custom destination**, and **Update a dashboard filter**, as the drill-through menu is only available to questions composed with the query builder.
+SQL questions will only have the option to **Go to a custom destination**, and **Update a dashboard filter**.
```

Comment:
Why drop this explanation?

Reply:
Because it doesn't really make sense as an explanation anymore: we do have limited drill-through menu now, but this option is still unavailable. Maybe I can say something with the word "limited"...